tests/httptrace/test_header_scrubber.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from pyctuator.httptrace.http_header_scrubber import scrub_header_value
+
+
+@pytest.mark.parametrize("key,value", [("Authorization", "Bearer 123"), ("authorization", "Bearer 123"), ("X-Csrf-Token", "foo"), ("authentication", "secret123")])
```

Comment:
`test_scrubbing` fails when the key is `authentication` (https://github.com/SolarEdgeTech/pyctuator/pull/45/checks#step:7:2482) because `_keys_to_scrub` doesn't include `authentication`.

Looking at https://developer.mozilla.org/en-US/docs/Web/HTTP/Authentication I think that we do want to include `.*authentication.*` in the list of headers that needs to be scrubbed.